clients/drcachesim/CMakeLists.txt

```diff
@@ -966,6 +966,22 @@ if (BUILD_TESTS)
       use_DynamoRIO_extension(tool.drcacheoff.burst_traceopts drcovlib_static)
     endif (NOT RISCV64)
 
+    if (NOT RISCV64) # TODO i#3544: Port tests to RISC-V 64
+      add_executable(tool.drcacheoff.windows-timestamps
+        "${CMAKE_CURRENT_SOURCE_DIR}/tests/window_test.cpp")
+      configure_DynamoRIO_static(tool.drcacheoff.windows-timestamps)
+      use_DynamoRIO_static_client(tool.drcacheoff.windows-timestamps drmemtrace_static)
+      target_link_libraries(tool.drcacheoff.windows-timestamps drmemtrace_raw2trace
+        drmemtrace_analyzer test_helpers)
+      if (WIN32)
+        # Just like for burst_replace, linking everything together takes effort.
+        target_link_libraries(tool.drcacheoff.windows-timestamps ${static_libc})
+      endif ()
+      set(tool.drcacheoff.windows-timestamps_uses_configure ON) # Avoid dup DEBUG.
```

Comment:
Please see the top-level comment about removing this (even if it doesn't affect the popups, it still should not be here and will affect any DEBUG use in the main test file).  Making a comment so it can be marked whether resolved.

Reply:
Ack. Removed it.